brewtils/rest/easy_client.py

```diff
@@ -705,6 +711,47 @@ def find_jobs(self, **kwargs):
         """
         return self.client.get_jobs(**kwargs)
 
+    @wrap_response(
+        parse_method="parse_job_definitions", parse_many=False, default_exc=FetchError
+    )
+    def export_jobs(self, job_id_list):
+        # type: (Optional[JobIDList]) -> JobDefinitionList
+        """Export job definitions from an optional job ID list.
+
+        If `job_id_list` is None or empty, definitions for all jobs
```

Comment:
This interface feels weird to me, specifically the fact that I have to explicitly supply `None` in order to get **all** of the jobs. I think it would feel more natural if `job_id_list` were optional, and not supplying it at all is what gets me all the things.  So it would look like:

```python
all_jobs = easy_client.export_jobs()
some_jobs = easy_client.export_jobs(["6113e42c5ab6253f46838209", "6113dfbc5ab6253f46838205"])  # see earlier question about the necessity of JobIDList
```

Reply:
That was an error. The signature was meant to include `job_id_list=None`.